### PR TITLE
Odim 191 - updated docs/readme, deployment readme, and and individual readme files

### PR DESCRIPTION
**Pull request by chinmayic**
**Service-wise individual readme files:**
1. Formatting and semantic changes.
2. Added connection methods GET content in svc-aggregation/Readme.

**Deployment Readme file:**
Semantic changes.

**docs/Readme file:**
1. Added connection methods GET content.
2. Updated list of APIs.
3. Added TOC for connection methods content.
4.  Fixed these issues:
    a. The odimra user should not be a pre-created user with Ubuntu vm.
    b.  [Documentation] Unclear with the statement in document README.md file - statement at line no 125 to 127.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,14 +124,25 @@ To install `Ubuntu Make`, run the following command:
...
     Ensure that the above ports are not in use.
+  - The following users are created and added to group ids automatically when the certificates are generated during deployment. 
+  
+    |User Id| Group Id|
+	-----|---------|
+	|`odimra`|1234 |
+	|`plugin`|1235 |
+	
+    `odimra` is created on both the VM and the container for the resource aggregator.
+	`plugin` is created  on both the VM and the container for the GRF plugin.
```
Comment (shivacharanms) [inline README.md:143]: Indentation correction needed

**Review by rahulroshan-kachchap — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chinmayic) [reply README.md]: Addressed.

File: svc-events/README.md [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
 An event is a way of asynchronously notifying the client of some significant state change or error condition, usually of a time critical nature.
 
 Use these APIs to subscribe a northbound client to southbound events by creating a subscription entry in the service.
 
+**Supported endpoints**
+
+|API URI|Operation Applicable|Required privileges|
+|-------|--------------------|-------------------|
+|/redfish/v1/EventService|GET|`Login` |
+|/redfish/v1/EventService/Subscriptions|POST, GET|`Login`, `ConfigureManager` |
```
Comment (irappa-pattar) [inline svc-events/README.md:14]: Please recheck the required privileges for the endpoints

File: svc-events/README.md [hunk 3]
```diff
@@ -297,49 +300,49 @@ Only a user with `ConfigureComponents` privilege is authorized to create event s
...
 |EventTypes|Array \(string \(enum\)\)|Read-only \(Optional\)<br> |The types of events that are sent to the destination. For possible values, see "Event types" table.|
-|Types|Array \(string, null\)|Read-only \(Optional\)<br> |The list of  type values \(Schema names\) that correspond to the `Origins`. For possible values, perform `GET` on `redfish/v1/EventService` and check values listed under `Types` in the JSON response.<br> Examples: "ComputerSystem", "Storage", "Task"<br> |
+|ResourceTypes|Array \(string, null\)|Read-only \(Optional\)<br> |The list of resource type values \(Schema names\) that correspond to the `OriginResources`. For possible values, perform `GET` on `redfish/v1/EventService` and check values listed under `ResourceTypes` in the JSON response.<br> Examples: "ComputerSystem", "Storage", "Task"<br> |
 |Context|String|Read/write Required \(null\)<br> |A string that is stored with the event destination subscription.|
 |MessageId|String|Read-only \(Optional\)<br> |The key used to find the message in a Message Registry.|
```
Comment (irappa-pattar) [inline svc-events/README.md:305]: MessageId is `MessageIds` and its an array

File: svc-events/README.md [hunk 4]
```diff
@@ -297,49 +300,49 @@ Only a user with `ConfigureComponents` privilege is authorized to create event s
...
+|SubordinateResources|Boolean|Read-only \(null\)|Indicates whether the service supports the `SubordinateResource` property on event subscriptions or not. If it is set to `true`, the service creates subscription for an event originating from the specified `OriginResoures` and also from its subordinate resources. For example, by setting this property to `true`, you can receive specified events from a compute node: `/redfish/v1/Systems/{ComputerSystemId}` and from its subordinate resources such as:<br> `/redfish/v1/Systems/{ComputerSystemId}/Memory`,<br> `/redfish/v1/Systems/{ComputerSystemId}/EthernetInterfaces`,<br> `/redfish/v1/Systems/{ComputerSystemId}/Bios`,<br> `/redfish/v1/Systems/{ComputerSystemId}/Storage`|
+|OriginResources|Array| Required \(null\)<br> |Resources for which the service only sends related events. If this property is absent or the array is empty, events originating from any resource will be sent to the subscriber. For possible values, see "Origin resources" table.|
```
Comment (irappa-pattar) [inline svc-events/README.md:310]: OriginResources is optional parameter

File: svc-events/README.md [hunk 5]
```diff
@@ -297,49 +300,49 @@ Only a user with `ConfigureComponents` privilege is authorized to create event s
...
+|/redfish/v1/Systems|All computer system resources available in Resource Aggregator for ODIM for which the service sends only related events. By setting `EventType` property in the request payload to `ResourceAdded` or `ResourceRemoved` and `OriginResources` property to `/redfish/v1/Systems`, you can receive notifications when a system is added or removed in Resource Aggregator for ODIM.|
+|/redfish/v1/Chassis|All chassis resources available in Resource Aggregator for ODIM for which the service sends only related events.|
+|/redfish/v1/Fabrics|All fabric resources available in Resource Aggregator for ODIM for which the service sends only related events.|
+|/redfish/v1/TaskService/Tasks|All tasks scheduled by or being executed by Redfish `TaskService`. By subscribing to Redfish tasks, you can receive task status change notifications on the subscribed destination client.<br> By specifying the task URIs as `OriginResources` and `EventTypes` as `StatusChange`, you can receive notifications automatically when the tasks are complete.<br> To check the status of a specific task manually, perform HTTP `GET` on its task monitor until the task is complete.<br> |
```
Comment (irappa-pattar) [inline svc-events/README.md:321]: `/redfish/v1/Managers` also supported collection

File: svc-events/README.md [hunk 6]
```diff
@@ -458,7 +473,7 @@ curl -i POST \
    "EventTimestamp":"{Event_Time_Stamp}",
    "EventType":"{Event_Type_String}",
    "Message":"{Message_String}",
-   "Types":[{_type_string}],
+   "ResourceTypes":[{resource_type_string}],
```
Comment (irappa-pattar) [inline svc-events/README.md:476]: ResourceTypes is not required in submittestevent

File: svc-events/README.md [hunk 7]
```diff
@@ -494,7 +503,7 @@ curl -i POST \
    "EventTimestamp":"2020-02-17T17:17:42-0600",
    "EventType":"Alert",
    "Message":"The LAN has been disconnected",
-   "Types":["EthernetInterface"],
+   "ResourceTypes":["EthernetInterface"],
```
Comment (irappa-pattar) [inline svc-events/README.md:506]: ResourceTypes is not required in submittestevent

File: svc-events/README.md [hunk 8]
```diff
@@ -517,13 +526,13 @@ curl -i POST \
 |EventTimestamp|String|Optional|The date and time stamp for the event to add. When the event is received, it translates as the time the event occurred.|
 |EventType|String \(enum\)|Optional|The type for the event to add. For possible property values, see "EventType" in [Creating an event subscription](#creating-an-event-subscription).|
 |Message|String|Optional|The human-readable message for the event to add.|
-|Types|Array \(string, null\)|Read-only  <br> |The list of  type values \(Schema names\) that correspond to `OriginOfCondition`. For possible values, perform `GET` on `redfish/v1/EventService` and check values listed under `Types` in the JSON response.<br> Examples: "ComputerSystem", "Storage", "Task"<br> |
+|ResourceTypes|Array \(string, null\)|Read-only  <br> |The list of resource type values \(Schema names\) that correspond to `OriginOfCondition`. For possible values, perform `GET` on `redfish/v1/EventService` and check values listed under `ResourceTypes` in the JSON response.<br> Examples: "ComputerSystem", "Storage", "Task"<br> |
```
Comment (irappa-pattar) [inline svc-events/README.md:529]: ResourceTypes is not required in submittestevent

**Review by irappa-pattar — CHANGES_REQUESTED** [review]
please address the comments

File: svc-events/README.md (hunk 2 above)
Comment (chinmayic) [reply svc-events/README.md]: I have rechecked and made changes accordingly.

**Review by irappa-pattar — APPROVED** [review]

File: README.md [hunk 9]
```diff
@@ -122,14 +124,27 @@ To install `Ubuntu Make`, run the following command:
...
     Ensure that the above ports are not in use.
+  - The following users are created and added to group ids automatically when the certificates are generated during deployment. 
+  
+    |User Id| Group Id|
+	-----|---------|
+	|`odimra`|1234 |
+	|`plugin`|1235 |
+	
+
+     `odimra` is created on both the VM and the container for the resource aggregator.
```
Comment (Bharath-KKB) [inline README.md:143]: VM ? We are developing this for k8s which is primarily deployed on BM. Docker images could run on either. So remove references to VM. What is the Group Id ?

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
answer queries

**Review by Bharath-KKB — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**